go.mod

```diff
@@ -54,7 +54,7 @@ require (
 	github.com/elazarl/goproxy v0.0.0-20190630181448-f1e96bc0f4c5 // indirect
 	github.com/elazarl/goproxy/ext v0.0.0-20190630181448-f1e96bc0f4c5 // indirect
 	github.com/emicklei/go-restful v2.9.6+incompatible // indirect
-	github.com/envoyproxy/go-control-plane v0.9.6-0.20200623200256-ba7954ce7df4
+	github.com/envoyproxy/go-control-plane v0.9.7-0.20200728064350-8b7c7dde6597
```

Comment:
I don't see `v0.9.7` release anywhere here https://github.com/envoyproxy/go-control-plane/tags. Is it a pre-relase or something?

Reply:
I was experimenting with latest sha